mod_reforged/hooks/factions/faction.nut

```diff
@@ -0,0 +1,7 @@
+::Reforged.HooksMod.hook("scripts/factions/faction", function (q) {
+	// Overwrite, because we only want to hide certain settlements at 50 relation, if they didn't have a recent relationship change with the player
+	q.isHidden = @() function()
+	{
+		return this.m.IsHidden || (this.m.IsHiddenIfNeutral && this.m.PlayerRelation == 50 && this.m.PlayerRelationChanges.len() == 0);
```

Comment:
Looks good to me but I have a question. Vanilla here does a `Math.abs(this.m.PlayerRelation - 50.0) == 0.0` comparison whereas you directly check for `this.m.PlayerRelation == 50`. Do we know if there is a good reason vanilla did that `abs` thing and if there are any pitfalls with this new method? Btw the return type of `::Math.abs` is integer.

Reply:
I don't understand vanillas way of doing the check. 
Maybe this was a leftover from a refactoring from an earlier version.

Reply:
Ok let's go with your method. If anything strange happens we will see.